test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/CacheCommand.cs

```diff
@@ -83,7 +85,7 @@ private string BuildArgs()
                 FrameworkVersionOption);
         }
 
-        private string ProfileProjectOption => string.IsNullOrEmpty(_profileProject) ? "" : $"--entries {_profileProject}";
+        private string ProfileProjectOption => _profileProject.Count == 0  ? "" :  string.Join(" ", _profileProject) ;
```

Comment:
These comments are repeated for the PublishCommand.cs file.

---
In reply to: [104685061](https://github.com/dotnet/cli/pull/5942#discussion_r104685061) [](ancestors = 104685061,104684104)